raiden_libs/transport/transport.py

```diff
@@ -19,12 +21,12 @@ class Transport(gevent.Greenlet):
     """
     def __init__(self):
         super().__init__()
-        self.message_callbacks = list()
+        self.message_callbacks = defaultdict(list)
 
-    def add_message_callback(self, callback):
-        self.message_callbacks.append(callback)
+    def add_message_callback(self, callback, target_node: str = None):
+        self.message_callbacks[target_node].append(callback)
 
-    def run_message_callbacks(self, data):
+    def run_message_callbacks(self, data, target_node: str = None):
```

Comment:
maybe update this with a simple broadcast option: If `target_node is None`, deliver message to all nodes.